device_samples/getting_started/src/connections/connection_string.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+// Choose a protocol by uncommenting one of these transports.
+const Protocol = require('azure-iot-device-mqtt').Mqtt;
+// const Protocol = require('azure-iot-device-amqp').Amqp;
+// const Protocol = require('azure-iot-device-http').Http;
+// const Protocol = require('azure-iot-device-mqtt').MqttWs;
+// const Protocol = require('azure-iot-device-amqp').AmqpWs;
+
+const Client = require('azure-iot-device').Client;
+const Message = require('azure-iot-device').Message;
+
+// String containing Hostname, Device Id & Device Key in the following formats:
+//  "HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>"
+const deviceConnectionString = process.env.IOTHUB_DEVICE_CONNECTION_STRING;
+
+// make sure we have a connection string before we can continue
+if (deviceConnectionString === '' || deviceConnectionString === undefined) {
+  console.error('\x1b[31m%s\x1b[0m', 'Missing device connection string');
+  process.exit(0);
+}
+
+function generateMessage() {
+  const data = JSON.stringify({
+    deviceId: 'my-first-device',
+    temperature: 20 + Math.random() * 10,
+  });
+  
+  return new Message(data);
+}
```

Comment:
nit: this function is copy/pasted all over our samples.  We could probably delete hundreds of lines if we just hardcode messages data.

Reply:
I fixed this in the connections samples, but it looks like it still adds value in the send messages samples. I guess we could simplify the telemetry we send. that would make it easier to remove that function and just hard code.

@BertKleewein  thought?